packages/edit-site/src/components/layout/hooks.js

```diff
@@ -5,15 +5,9 @@ import { useEffect, useState } from '@wordpress/element';
 import { useSelect } from '@wordpress/data';
 import { store as coreStore } from '@wordpress/core-data';
 
-/**
- * Internal dependencies
- */
-import useEditedEntityRecord from '../use-edited-entity-record';
-
 const MAX_LOADING_TIME = 10000; // 10 seconds
 
 export function useIsSiteEditorLoading() {
-	const { isLoaded: hasLoadedPost } = useEditedEntityRecord();
```

Comment:
Are we sure that this won't change the `useIsSiteEditorLoading` hooks behavior?

Reply:
For me, yes because the retrievable of the entity is one of the resolvers anyway.